### PR TITLE
Update appstream meatadata to current fromat

### PR DESCRIPTION
**Pull request by innir**
Hi,

I updated the appstream metadata the current format, following:

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

Best,
Philip

### CODE REVIEW

**Comment by innir** [comment]
Sorry for the messy commits -- I can redo them if you want a cleaner changelog

File: share/metainfo/org.kitone.subtitleeditor.appdata.xml [hunk 1]
```diff
@@ -13,16 +14,18 @@
...
-		<keywords>
-			<keyword>subtitles</keyword>
-			<keyword>subtitle</keyword>
-			<keyword>captions</keyword>
-			<keyword>video</keyword>
-		</keywords>
-</application>
+	<screenshots>
+		<screenshot type="default">
+			<image>http://home.gna.org/subtitleeditor/screenshots/se-0.34-01.png</image>
```
Comment (kitone) [inline share/metainfo/org.kitone.subtitleeditor.appdata.xml]: You change the url with the old one
https://kitone.github.io/subtitleeditor/docs/screenshoot.png

**Comment by innir** [comment]
ups, sorry, will change that.

File: share/metainfo/org.kitone.subtitleeditor.appdata.xml (hunk 1 above)
Comment (innir) [inline share/metainfo/org.kitone.subtitleeditor.appdata.xml]: fixed it now

**Comment by kitone** [comment]
Thank you Philip!

Best regards,
kitone

**Pull request merged**